Engine/INotifyFieldChanged.cs

```diff
@@ -4,6 +4,6 @@ namespace Engine
 {
     public interface INotifyFieldChanged
     {
-        public abstract void OnFieldChanged(FieldInfo fieldInfo);
+        public void OnInspectorFieldChanged(FieldInfo fieldInfo);
```

Comment:
Nothing related to editor should exist in engine, rename back to OnFieldChanged

Reply:
++